Doc/using/configure.rst

```diff
@@ -945,6 +945,17 @@ See :source:`Mac/README.rst`.
    Specify the name for the python framework on macOS only valid when
    :option:`--enable-framework` is set (default: ``Python``).
 
+.. option:: --with-app-store-compliance
+.. option:: --with-app-store-compliance=PATCH-FILE
+
+   The Python standard library contains strings that are known to trigger
+   automated inspection tool errors when submitted for distribution by
+   the macOS and iOS App Stores. If enabled, this option will apply the list of
+   patches that are known to correct app store compliance. A custom patch
+   file can also be specified. This option is disabled by default.
+
+   .. versionadded:: 3.13
```

Comment:
Is this going to backported to 3.12 also? There's a `3.12` label attached to the PR :)

Reply:
See discussion above. The *issue* exists in 3.12; the suggestion was that the back port decision was one for @Yhg1s as 3.12 release manager, since it's not a bug fix in the strictest sense, and changes the configuration options.

Reply:
for reference: [this comment](https://github.com/python/cpython/pull/120984#issuecomment-2188214941) and [this one](https://github.com/python/cpython/pull/120984#issuecomment-2190954638).